manifests/profile/cache_rpms.pp

```diff
@@ -15,13 +19,23 @@
     timeout   => '600',
     logoutput => false,
     require   => Yumrepo['epel'],
```

Comment:
We probably could/should fix it, but I think it would require some more complexity. My point was just that we can't add a "before" to the epel resource declaration because it's declared in multiple places.